src/TaylorIntegration.jl

```diff
@@ -29,20 +29,20 @@ function __init__()
 end
 
 
-@inline function diffeq!(a::Taylor1{T}, b::Taylor1{T}, k::Int) where {T<:TaylorSeries.NumberNotSeries}
+@inline function ode!(a::Taylor1{T}, b::Taylor1{T}, k::Int) where {T<:TaylorSeries.NumberNotSeries}
```

Comment:
What about `solcoeff!` or something alike?

Reply:
Done; thanks for the suggestion!